APITesters/SwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -125,8 +124,6 @@ private func checkPurchasesPurchasingAPI(purchases: Purchases) {
     purchases.checkTrialOrIntroductoryPriceEligibility([String](), completion: checkEligComplete)
     purchases.checkTrialOrIntroductoryPriceEligibility([String]()) { _ in }
 
-    purchases.paymentDiscount(forProductDiscount: productDiscount, product: skp) { _, _ in }
```

Comment:
i think in the past we've left APITester calls in for deprecated methods, since technically they are still part of our public API. 

i always just added a comment saying something like "should have deprecation warning", but there's definitely a better way to do it. maybe we could make a method for testing all deprecated apis? idk if there's any way to confirm the warning appears as we expect..

Reply:
oh, nvm... seems like we are obsoleting, not deprecating?

Reply:
Oh yeah when I said deprecated I meant obsoleted. I keep mixing both words 😇